### PR TITLE
SCP - 5000 and Outfits

### PR DESCRIPTION
**Pull request by tichys**
Its SCP - 5000 and its a fucking ninja suit, what do you want from me? a cookie?

### CODE REVIEW

**Comment by TheDarkElites** [comment]
Im going to kill you you didn't even separate your thing to a different branch from the net changes

**Comment by TheDarkElites** [comment]
AND HE TEST MERGED IT

**Comment by aidan-bowman** [comment]
> AND HE TEST MERGED IT

we truly live in a society

**Comment by tichys** [comment]
> Im going to kill you you didn't even separate your thing to a different branch from the net changes

hope about you do that >:C... its a small change and one pr that i didnt make a new branch on

File: code/modules/clothing/spacesuits/rig/suits/light.dm [hunk 1]
```diff
@@ -177,3 +177,45 @@
 		/obj/item/rig_module/stealth_field,
 		/obj/item/rig_module/vision
 		)
+
+
+/obj/item/rig/light/stealth/scp5000
```
Comment (TheDarkElites) [inline code/modules/clothing/spacesuits/rig/suits/light.dm]: Doesn't properly initialize SCP datum

for shame lestat for shame

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

**Comment by TheDarkElites** [comment]
pulling this off TM until it gets a proper review and and gets approved

**Comment by checkraisefold** [comment]
overwrote the rig module instead of making a subtype separate one
this smells

File: code/modules/clothing/spacesuits/rig/suits/light.dm [hunk 2]
```diff
@@ -177,3 +177,45 @@
...
+	name = "total exclusion harness"
+	suit_type = "stealth"
+	desc = "The Total Exclusion Harness is a suit produced by an unknown version of the SCP Foundation in a parallel universe. It is capable of rendering it's wearer entirely incomprehendable, and thus, invisible, whilst also maintaining their vital functions and carrying a portable datavault and AI interface."
+	icon_state = "stealth_rig"
+	online_slowdown = 0
+	cell_type =  /obj/item/cell/infinite
+	aimove_power_usage = 0
+	emp_protection = 100
+	siemens_coefficient = 1
+	req_access = list(ACCESS_SYNDICATE)
```
Comment (checkraisefold) [inline code/modules/clothing/spacesuits/rig/suits/light.dm]: you dont need this here it is already inherited from the scp5000 type

**Pull request merged**